packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_transmission.ts

```diff
@@ -1,9 +1,10 @@
-import type { IMaterial, IKHRMaterialsTransmission } from "babylonjs-gltf2interface";
+import { type IMaterial, type IKHRMaterialsTransmission, ImageMimeType } from "babylonjs-gltf2interface";

```

Comment:
cc @RaananW, I seem to recall that having the "type" inside the import does not work for some reasons (tool ?).

Reply:
I didn't do this. Vscode did it automatically. And yeah, it doesn't work with UMD for some reason.

Reply:
```suggestion
import type { IMaterial, IKHRMaterialsTransmission } from "babylonjs-gltf2interface";
import { ImageMimeType } from "babylonjs-gltf2interface";
```